src/components/Widgets/MarkDown/index.vue

```diff
@@ -94,7 +94,7 @@ export default {
       if (!content) return ''
 
       return DOMPurify.sanitize(content, {
-        ALLOWED_TAGS: ['p', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'ul', 'ol', 'li', 'strong', 'em', 'code', 'pre', 'blockquote', 'a'],
+        ALLOWED_TAGS: ['p', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'ul', 'ol', 'li', 'strong', 'em', 'code', 'pre', 'blockquote', 'a', 'img'],
         FORBID_TAGS: ['script', 'style', 'iframe', 'frame', 'object', 'embed'],
         FORBID_ATTR: ['onerror', 'onload', 'onclick', 'onmouseover']
       })
```

Comment:
Please upload this piece of code to see the difference between the given code snippet and any updated version released after September 2021, including changes in allowed HTML elements, attributes, or restrictions on external scripting elements such as scripts, frames, objects, embeds etc.